### ISSUE TITLE
[Content group simple]: QA: In iPhone with VO, after activating the video button the video player is not triggered

### ISSUE THREAD

**Issue by gopinathan-n** [carbon-design-system/carbon-for-ibm-dotcom#9656]
### Description

Page Element:
Image type - video

Issue Description:
In vo iPhone, After activating the video button the video player is not triggered. Instead the button disappears

Recommend Fix:
Video should get trigger. 
Note:Issue is inconsistent.

Tool:
Voice Over_iPhone

### Component(s) impacted

Content group simple

### Browser

Safari

### Carbon for IBM.com version

v1.40.0-rc.0

### Severity

Severity 4 = The problem is not visible to or noticeable to an average user. Affects minor functionality, no workaround needed.

### Application/website

Carbon for IBM.com

### Package

@carbon/ibmdotcom-web-components

### CodeSandbox example

https://carbon-design-system.github.io/carbon-for-ibm-dotcom/canary/web-components/?path=/story/components-content-block-media--default

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/carbon-design-system/carbon-for-ibm-dotcom/blob/main/.github/CODE_OF_CONDUCT.md)
- [ ] I checked the [current issues](https://github.com/carbon-design-system/carbon-for-ibm-dotcom/issues) for duplicate issues
